config/application.yml.default

```diff
@@ -107,7 +107,7 @@ good_job_queues: 'default:5;low:1;*'
 good_job_queue_select_limit: 5_000
 gpo_designated_receiver_pii: '{}'
 hide_phone_mfa_signup: false
-identity_pki_disabled: false
+identity_pki_disabled: true
```

Comment:
is this a local change we also want to remove?
```suggestion
identity_pki_disabled: false
```

Reply:
Sorry!  'twas accidental...